src/parsers/helper.ts

```diff
@@ -9,14 +9,18 @@
 import {ParserOptions, ParserPlugin} from '@babel/parser';
 
 /**
- * determine if the file is a typescript (ts), javascript (js) otherwise returns undefined.
+ * determine if the file is a javascript (js), typescript (ts), or typescript JSX (tsx),
+ * otherwise returns undefined.
  * @param filepath
- * @returns 'js'|'ts' or undefined
+ * @returns 'js'|'ts'|'tsx' or undefined
  */
-export const supportedFileType = (filePath: string): 'ts' | 'js' | undefined => {
-  if (filePath.match(/\.tsx?$/)) {
+export const getFileType = (filePath: string): 'js' | 'ts' | 'tsx' | undefined => {
+  if (filePath.match(/\.ts$/)) {
```

Comment:
I like your comment on the other thread about making it case-insensitive...

Reply:
Oops, lol. Forgot about that